### PR TITLE
translated the message on delete modal

### LINKED ISSUE #193: Message on note delete
The message on note(s) delete is always in English, irrespective of chosen language. Please, refer to screenshots:

01_delete_note
02_delete_note
03_delete_note

### PR DESCRIPTION
**Pull request by AliYar-Khan**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [X] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
The message note was not being translated into the language selected. This PR fixes that.
## Featured Covered in this PR

- [X] Translate the message on delete note to all available languages

## Related Tickets & Documents

- Related Issue #
- Closes #193 

## Screenshots, Recordings
image

*Please replace this line with instructions on how to test your changes, or any screenshots or recording you can attach here.*

## Tested Feature??

- [X] In Real Device.
- [ ] In Emulator

### CODE REVIEW

**Comment by SankethBK** [comment]
Hey thanks for contributing!

But the files you have modified are actually auto-generated files. We need to add them in `lib/l10n/intl_<language-code.arb` files and install [flutter-intls ](https://marketplace.visualstudio.com/items?itemName=localizely.flutter-intl) VSCode extension which will auto-generate these files. You can refer this [comment](https://github.com/SankethBK/diaryvault/issues/75#issue-1932556975)

**Comment by AliYar-Khan** [comment]
okay I will fix it and get back to you

**Comment by AliYar-Khan** [comment]
@SankethBK  I have added the translations in arb files. Now how it will auto generate the code?

**Comment by SankethBK** [comment]
install [flutter-intls ](https://marketplace.visualstudio.com/items?itemName=localizely.flutter-intl)VSCode extension which will auto-generate these files.

**Comment by AliYar-Khan** [comment]
I tried it but having issue as i am not able to reference the variables

image

**Comment by SankethBK** [comment]
Push your arb file changes to GitHub, I'll generate them in my lical

**Comment by AliYar-Khan** [comment]
@SankethBK I tried this as well 
```
fvm flutter gen-l10n
```

```
Because l10n.yaml exists, the options defined there will be used instead.
To use the command line arguments, delete the l10n.yaml file in the Flutter project.


"ar": 40 untranslated message(s).
"de": 15 untranslated message(s).
"es": 15 untranslated message(s).
"fi": 31 untranslated message(s).
"fr": 15 untranslated message(s).
"gu": 31 untranslated message(s).
"he": 40 untranslated message(s).
"hi": 15 untranslated message(s).
"id": 28 untranslated message(s).
"kn": 40 untranslated message(s).
"pa": 40 untranslated message(s).
"pl": 2 untranslated message(s).
"pt": 15 untranslated message(s).
"sw": 40 untranslated message(s).
"te": 14 untranslated message(s).
"zh": 15 untranslated message(s).
To see a detailed report, use the untranslated-messages-file
option in the l10n.yaml file:
untranslated-messages-file: desiredFileName.txt
<other option>: <other selection>


This will generate a JSON format file containing all messages that
need to be translated.
```

**Comment by AliYar-Khan** [comment]
@SankethBK do let me know what I was doing wrong. Purpose of contributing is to learn !! Thanks

**Comment by AliYar-Khan** [comment]
@SankethBK Hi what is the progress ?

File: lib/core/widgets/home_page_app_bar.dart [hunk 1]
```diff
@@ -547,7 +547,7 @@ class DeleteIcon extends StatelessWidget {
                     mainAxisSize: MainAxisSize.min,
                     children: [
                       Text(
-                        "You are about to delete $deletionCount  item${deletionCount > 1 ? "s" : ""}",
+                        "${S.current.aboutToDelete} $deletionCount ${S.current.item}${deletionCount > 1 ? "s" : ""}",
                         style: TextStyle(
```
Comment (SankethBK) [inline lib/core/widgets/home_page_app_bar.dart:551]: `${S.current.item}${deletionCount > 1 ? "s" : ""}",` this part wouldn't be meaningful for other languages. The purpose of this was to display `item` if the count is 1 and `items` if the count > 1, but in other languages it won't be `s` at the end for plural of item. I'll think about how to handle this, let me know if you have any solution

**Comment by SankethBK** [comment]
Hey, sorry missed your last comment. I have a added a comment related to plurals, I'll get back on it.

File: lib/core/widgets/home_page_app_bar.dart (hunk 1 above)
Comment (SankethBK) [reply lib/core/widgets/home_page_app_bar.dart:551]: Hi, let's keep two different strings for `item` and `items`

`${S.current.aboutToDelete} $deletionCount ${deletionCount > 1 ? S.current.item : S.current.items}`

Comment (AliYar-Khan) [reply lib/core/widgets/home_page_app_bar.dart:551]: yeah I agreed

**Review by marek-jakub — APPROVED** [review]
Thank you for sorting out the deletion message.

**Pull request closed without merging**